adding_species_manuscript.tex

```diff
@@ -71,35 +71,36 @@
 \author[6]{Victoria Caudill}
 \author[9]{Jean Cury}
 \author[10]{Ignacio Echevarria}
-\author[11,12]{Ahmed Hasan}
-\author[13,14]{Xin Huang}
-\author[15]{Leonardo Nicola Martin Iasi} % lastname Iasi
-\author[16]{Jana Obšteter}
-\author[17]{Vitor Antonio Corrêa Pavinato} % lastname Pavinato
-\author[18,19]{David Peede}
-\author[20]{Ekaterina Noskova}
-\author[21,22]{Alice Pearson}
-\author[23]{Manolo Perez}
+\author[11]{Benjamin C. Haller}
+\author[12,13]{Ahmed Hasan}
+\author[14,15]{Xin Huang}
+\author[16]{Leonardo Nicola Martin Iasi} % lastname Iasi
+\author[17]{Jana Obšteter}
+\author[18]{Vitor Antonio Corrêa Pavinato} % lastname Pavinato
+\author[19,20]{David Peede}
+\author[21]{Ekaterina Noskova}
+\author[22,23]{Alice Pearson}
+\author[24]{Manolo Perez}
 \author[6]{Murillo F. Rodrigues}
 \author[6]{Chris C. R. Smith}
-\author[24]{Jeff Spence}
+\author[25]{Jeff Spence}
 \author[6]{Anastasia Teterina}
 \author[6]{Silas Tittes}
-\author[25]{Per Unneberg}
-\author[26]{Juan Manuel Vasquez}
-\author[27]{Ryan Waples}
-\author[28]{Anthony Wilder Wohns}
-\author[29]{Yan Wong}
-\author[30]{Reed Cartwright}
-\author[31]{Aaron P. Ragsdale}
-\author[32]{Franz Baumdicker}
-\author[33]{Gregor Gorjanc}
-\author[34]{Ryan N. Gutenkunst}
-\author[29]{Jerome Kelleher}
+\author[26]{Per Unneberg}
+\author[27]{Juan Manuel Vasquez}
+\author[28]{Ryan Waples}
+\author[29]{Anthony Wilder Wohns}
+\author[210]{Yan Wong}
```

Comment:
this should be \author[30], right?

Reply:
ack what happened there

Reply:
oh I know what happened, heh - I asked vim to increment all the numbers but had only the last digit highlighted. =)